README.md

```diff
@@ -75,6 +75,26 @@ var results = conn.Execute(sql.CommandText, sql.Parameters);
 var results = syntax.QueryCollection<EmployeeObj>(list);
 ```
 
+# Constructing a Query
+
+Searchlight supports most operators common to SQL, such as:
+* Equals (=, EQ)
+* Greater Than (>, GT)
+* Greater Than Or Equal (>=, GE)
+* Less Than (<, LT)
+* Less Than Or Equal (<=, LE)
+* Not Equal (!=, NE, <>)
+* In
+* Contains
+* StartsWith
+* EndsWith
+* IsNull
+* AND
+* OR
+* NONE  
```

Comment:
I'm not sure we support "NONE"?

Reply:
I think I read the file wrong, it's in our Enums but we don't support it in a filter so I took it off